example/android/app/build.gradle

```diff
@@ -143,6 +143,12 @@ android {
 
 dependencies {
 
+
+
+
+
+
+
```

Comment:
expo prebuild just generated a bunch of empty lines... I we leave this here otherwise every build will have this diff?

Reply:
yeah that's fine, this is basically autogenerated code.